packages/node/src/node.ts

```diff
@@ -1 +1,113 @@
-export default class Node {}
+import * as cf from "@counterfactual/cf.js";
+import { EventEmitter } from "events";
+import * as _ from "lodash";
+
+import { MultisigAddress, StateChannelInfo } from "./channel";
+
+export interface NodeOptions {
+  channels?: Map<MultisigAddress, StateChannelInfo>;
+}
+
+/**
+ * The Node class is the entry point to the @counterfactual/node package.
+ * It's responsible for:
+ * - encapsulating channels and their apps
+ * - connecting with a provided database to persistently store commitments
+ * - registering a signing service
+ */
+export class Node {
+  private channels: Map<MultisigAddress, StateChannelInfo> = new Map();
```

Comment:
should abstract this inside a database connection; can start with redis

Reply:
WIP on another branch